GMG/Scenes/Export/ExportIntent.swift

```diff
@@ -15,29 +15,34 @@ final class ExportIntent: ExportIntentProtocol {
         self.model = model
     }
 
-    func onAppear() {
-        model.readScore { score in
-            let renderer = ImageRenderer(content: ChordSheetView(score: score))
-
-            if let uiImage = renderer.uiImage {
-                model.updateSheetImage(uiImage)
-
-                if let data = uiImage.pngData() {
-                    let fileName = "sheet-\(score.id.uuidString).png"
-                    let url = FileManager.default
-                        .temporaryDirectory
-                        .appendingPathComponent(fileName)
-
-                    do {
-                        try data.write(to: url)
-                        model.updateSheetURL(url)
-                    } catch {
-                        print("error")
-                    }
+    func onAppear(_ score: Score) {
+
+        let renderer = ImageRenderer(content: ChordSheetView(score: score))
+
+        if let uiImage = renderer.uiImage {
+            model.updateSheetImage(uiImage)
+
+            if let data = uiImage.pngData() {
+                let fileName = "sheet-\(score.id.uuidString).png"
+                let url = FileManager.default
+                    .temporaryDirectory
+                    .appendingPathComponent(fileName)
+
+                do {
+                    try data.write(to: url)
+                    model.updateSheetURL(url)
+                } catch {
+                    print("error")
                 }
```

Comment:
if let data 보다

guard 구문을 사용하는 것이 현재 context에서는 적합한 것 같습니다.

data가 꼭 필요한 상황인 것 처럼 보여서요!

그리고 에러 구문을 좀 더 구체적으로 적어야할 것 같습니다! 

```suggestion
                guard let data = uiImage.pngData() else { // 에러 시나리오 }

                let fileName = "sheet-\(score.id.uuidString).png"
                let url = FileManager.default
                    .temporaryDirectory
                    .appendingPathComponent(fileName)

                do {
                    try data.write(to: url)
                    model.updateSheetURL(url)
                } catch {
                    // 에러 시나리오
                }
```

Reply:
레몬 PR에서 수정한 거 반영하기로 했습니다